src/h5web/visualizations/shared/utils.ts

```diff
@@ -144,6 +151,29 @@ export function getTicksProp(
     : { numTicks };
 }
 
+export function getTickFormatter(
+  domain: Domain,
+  availableSize: number,
+  scaleType: ScaleType
+): (val: number | { valueOf(): number }) => string {
+  if (scaleType !== ScaleType.Log) {
+    return TICK_FORMAT;
+  }
+
+  // If available size allows for all log ticks to be rendered without overlap, use default formatter
+  const threshold = adaptedLogTicksThreshold(availableSize);
+  if (Math.log10(domain[1]) - Math.log10(domain[0]) < threshold) {
```

Comment:
`domain` may be composed of two negative values making the issue of this uncertain.

Reply:
Good catch, should work by comparing the absolute of the difference, right? `Math.abs(Math.log10(domain[1]) - Math.log10(domain[0])) < threshold`

Reply:
Ah no, because the values are reversed...

Reply:
arf...